sql-cli/pyproject.toml

```diff
@@ -26,23 +26,35 @@ Home = "https://astronomer.io/"
 Source = "https://github.com/astronomer/astro-sdk/sql-cli"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<4.0.0"
+python = ">=3.8,<3.11"
 typer = {extras = ["all"], version = "^0.6.1"}
 networkx = "^2.8.7"
 Jinja2 = ">=2.10.1, <=3.1"
 python-frontmatter = "^1.0.0"
 PyYAML = ">5.4.1"
 apache-airflow = ">2.0"
 python-dotenv = "^0.21.0"
+astro-sdk-python = {path = "../python-sdk", extras = ["all"]}
```

Comment:
Remember to change this to "non-dev" before the release though

Reply:
yes, @kaxil . Created a quick ticket for this so that we don't forget: https://app.zenhub.com/workspaces/sql-cli-631f43da09a3a67f2c795913/issues/astronomer/astro-sdk/1067